### DISCUSSION TITLE
Make a station with more than just one module

### DISCUSSION THREAD

**Discussion by etxbct** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#62] [Q&A] [answered]
A station can be built for one standard, like MMM. By adding two extra modules it is adapted to a different standard, like FREMO.

My station has 2 dm adaptation modules, one at each end, to adapt it to FREMO.
In station these two extra modules cannot be added.

My station also have extra 1 dm modules serving as in/out signal modules wgich can be placed about 1-1.5 m before the station with other modules in between.

/Benny

**Comment by fjallemark** [comment, marked as answer]
When your station has two distinct variants, and these variants are for two different module standards, Its possible to enter it as two separate modules with *same* name but different *variant* because they have different properties and separate drawings. Technically, both these modules can refer to the same station, but this is not supported in the user interface yet. So I added an issue #63 to adress this. 

Another option is to add the two MMM&FREMO-adapter modules as separate modules, because they may also be used independently from Bilbo. Let say a number of Swedish MMM module owners should build a Swedish stretch, then the adapter modules will be used in the end of this stretch, which not may be in conjunction with the Bilbo Station. As a layout planner, I prefer this way, because the freedom of using the adapter modules anywhere they are need.

As you describe your signal modules, they should be entered as separate modules, because they can be placed independently of other modules.  Name them like *Bilbo Signal West* or similar to emphasise that it is signals to use with Bilbo station.

To summarise my recommendation:
- *Bilbo* as an MMM-module only.
- *MMM/FREMO adapters* as separate modules.
- *Signal modules* as separate modules.

This will probably also be easier to maintain over time. In your module documentation PDF you can describe how a planner can or should utilise your modules at a meeting.
